tasks/main.yml

```diff
@@ -189,6 +189,12 @@
     shell: grub2-mknetdir --net-directory=/var/lib/tftpboot
     when: not staticips and ppc64le
 
+  - name: fake pxelinux.0 for ppc64le
+    copy:
+      src: /var/lib/tftpboot/boot/grub2/powerpc-ieee1275/core.elf
+      dest: /var/lib/tftpboot/pxelinux.0
+    when: not staticips and ppc64le
+
```

Comment:
@sydefree this will need a rebase due to https://github.com/RedHatOfficial/ocp4-helpernode/pull/62.

Reply:
Can I just remove those line and push again ?

Reply:
Can I just remove those lines and push agin ?

Reply:
Yeah..Please fetch the latest code and then remove the above lines. Just add the the console changes